Changelog.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added conversions from `Extent2D` to `Extent3D` and `Rect2D` (#557)
```

Comment:
You'll have to introducce a new `### Added` heading under `## [Unreleased] - ReleaseDate
` though, this isn't part of `## [0.35.1] - 2022-01-18` that released yesterday :grimacing:

Reply:
D'oh! 🐑😬 I didn't notice that 0.35.1 had been released and wrongly assumed those changes were still in the unreleased section.